### DISCUSSION TITLE
Time Varying Beta

### DISCUSSION THREAD

**Discussion by erinclancey** [kingaa/pomp#215] [Q&A]
I would like to make Beta (transmission parameter) a function over time. I cannot figure out where exactly the information about Beta over time gets incorporated into the code in pomp. Do you have any suggestions for an efficient way to code time varying Beta in a pomp model? Thank you!

**Comment by kingaa** [comment]
You can code it into your basic model components (the pertinent ones can be functions of time) using either R functions or C snippets.  You can also use the covariates facility.  There are many examples of this, both in the package itself and in the **pomp** community.

**Comment by erinclancey** [comment]
I will work on figuring out how to put the function for B(t) directly in the Csnippet. 

Is it common to have errors with pomp if you use a time step of 1 (instead of for example 1/7)?

**Reply by kingaa** [reply]
I assume you are referring to the time-step of one of the rprocess components?  In any case, the numerical magnitude of the time-step itself should not be the source of any error in the **pomp** code (so long as it is positive).  If errors arise when you change the magnitude of the time-step, these must originate in your own code.
